Dockerfile

```diff
@@ -13,26 +13,12 @@ RUN apt-get -y install rsync
 # Install dos2unix to remove 'r' characters from scripts
 RUN apt-get -y install dos2unix
 
-RUN apt-get -y install ruby-full
-RUN gem install bundle
-RUN bundle
-RUN curl -L "http://www.jonof.id.au/files/kenutils/pngout-20200115-linux.tar.gz" | tar -xz -C /usr/local/bin --strip-components 2 --wildcards '*/amd64/pngout'
-
 # Sets the working directory for any RUN, CMD, ENTRYPOINT, COPY and ADD instructions that follow it in the Dockerfile
 # https://docs.docker.com/engine/reference/builder/#workdir
-WORKDIR /usr/src/app
+WORKDIR /editor
 
 # Copy remaining files
 COPY . .
 
-# Remove 'r' characters from the scripts (otherwise it won't execute)
-RUN dos2unix ./scripts/generate-example-images.sh ./scripts/vendor.sh ./scripts/version.sh
-
 # Run Yarn
-RUN yarn
-
-# Sets the container executable (ENTRYPOINT) as yarn and the default argument (CMD) as start
-# https://docs.docker.com/engine/reference/builder/#entrypoint
-# https://docs.docker.com/engine/reference/builder/#cmd
-ENTRYPOINT ["yarn"]
-CMD ["start", "--host", "0.0.0.0"]
+RUN yarn install
```

Comment:
Does this command need to change? Do you need to update the readme as well?

Reply:
Sorry, mistakenly created a pull request for the source project instead of the forked one.

Reply:
Ahh, no problem. Excited to hear that you are using the Vega editor. If you have any changes you want to contribute back, I look forward to your prs.